utils/build/docker/dotnet_http/parametric/Dockerfile

```diff
@@ -0,0 +1,39 @@
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0
+
+RUN apt-get update && apt-get install -y dos2unix --no-install-recommends
+RUN apt-get update && apt-get install -y curl
+USER root
+WORKDIR /app
+
+# ensure that the Datadog.Trace.dlls are installed from /binaries
+COPY install_ddtrace.sh query-versions.fsx binaries* /binaries/
+RUN dos2unix /binaries/install_ddtrace.sh
+RUN /binaries/install_ddtrace.sh
+
+# restore nuget packages
+COPY ["ApmTestApi.csproj", "nuget.config", "*.nupkg", "./"]
+RUN dotnet restore "./././ApmTestApi.csproj"
```

Comment:
```suggestion
RUN dotnet restore "./ApmTestApi.csproj"
```

Reply:
Is this file used at all? Can we just delete it?